conf/vsc_calcua.config

```diff
@@ -31,6 +44,16 @@ if ( host == "antwerpen" ) {
             }
         }
     }
+    // APPTAINER_CACHEDIR/SINGULARITY_CACHEDIR
+    if ( !apptainer_cachedir ) {
+        System.err.println("\nWARNING: APPTAINER_CACHEDIR/SINGULARITY_CACHEDIR environment variable was not found.\nPlease add the line 'export APPTAINER_CACHEDIR=\"\${VSC_SCRATCH}/apptainer/cache\"' to your ~/.bashrc file (or set it with sbatch or in your job script).\nUsing the default storage location of Singularity/Apptainer ~/.apptainer/cache/. Read more about why this should be avoided in the VSC docs: https://docs.vscentrum.be/software/singularity.html#building-on-vsc-infrastructure\n")
+        // TODO: optional exit out here.
```

Comment:
I would make a pipeline exit here to enforce user to set these variables. Better to have this than having their home dirs fill up, which is a lot harder to debug :grin:

Reply:
Done!